application_form/api/sales/views.py

```diff
@@ -70,6 +84,50 @@ def execute_lottery_for_project(request):
     return Response({"status": "success"}, status=status.HTTP_200_OK)
 
 
+@api_view(http_method_names=["GET"])
+@require_http_methods(["GET"])  # For SonarCloud
+@permission_classes([IsDrupalServer])
+@authentication_classes([DrupalAuthentication])
+def sold_apartments(request):
+    """
+    Returns ids of apartments sold during start_time and end_time
+    By default
+        start_time: timezone.now() - timedelta(hours=1)
+        end_time = timezone.now()
+    """
+    end_time_obj = timezone.now()
+    start_time_obj = timezone.now() - timedelta(
+        hours=settings.DEFAULT_SOLD_APARMENT_TIME_RANGE
+    )
```

Comment:
Of course does not matter in practice at all, but to my engineer eye, it would be very pleasing if this was `start_time_obj - end_time_obj ...` so that the microseconds would also match 😄

Reply:
Yeah, good point, changed that